include/core/parameters.h

```diff
@@ -1291,41 +1291,87 @@ namespace Parameters
     declare_default_entry(ParameterHandler &prm, unsigned int index);
     void
     parse_parameters(ParameterHandler &prm);
-
-    unsigned int                 nb;
-    unsigned int                 order;
-    unsigned int                 initial_refinement;
-    double                       inside_radius;
-    double                       outside_radius;
-    bool                         time_extrapolation_of_refinement_zone;
+    // Number of ib particles declared
```

Comment:
It could improve readability to make parameter ''paragraphs'' and add line jumps between categories.

Reply:
I split them into group that follow the same structure as in the prm file